tools/dockerfile/push_testing_images.sh

```diff
@@ -67,6 +67,22 @@ ALL_DOCKERFILE_DIRS=(
 
 CHECK_FAILED=""
 
+if [ "${CHECK_MODE}" != "" ]
+then
+  # Check that there are no stale .current_version files (for which the corresponding
+  # dockerfile_dir doesn't exist anymore).
+  for CURRENTVERSION_FILE in $(find tools/ third_party/rake-compiler-dock -name '*.current_version')
+  do
+    DOCKERFILE_DIR="$(echo ${CURRENTVERSION_FILE} | sed 's/.current_version$//')"
```

Comment:
I think you need to escape the `.`, or it'll match any char. `sed 's/\.current_version$//'`

Reply:
Done.